parsing_engines/mappings.py

```diff
@@ -1,37 +1,31 @@
-COLUMN_NAME_MAPPING = {
+"""
+TODO: Mappings can all be defined here! No need for separate files. 
+"""
+
+GENERIC_COLUMN_MAPPING = {
     "time": "time/s",
     "Time": "time/s",
-    "time/s": "time/s",
-    "mode": "mode",
-    "ox/red": "ox/red",
-    "error": "error",
-    "control changes": "control changes",
-    "Ns changes": "Ns changes",
     "counter inc.": "counter inc",
-    "Ns": "Ns",
-    "I Range": "I Range",
-    "control/V/mA": "control/V/mA",
-    "Ecell/V": "Ecell/V",
-    "I/mA": "I/mA",
     "dq/mA.h": "dq/mA_h",
     "(Q-Qo)/mA_h": "(Q-Qo)/mA.h",
     "Energy/W_h": "Energy/W.h",
-    "Analog IN 1/V": "Analog IN 1/V",
-    "Analog OUT/V": "Analog OUT/V",
     "Energy charge/W_h": "Energy charge/W.h",
     "Energy discharge/W_h": "Energy discharge/W.h",
-    "Capacitance charge/µF": "Capacitance charge/µF",
-    "Capacitance discharge/µF": "Capacitance discharge/µF",
-    "x": "x",
     "Q discharge/mA_h": "Q discharge/mA.h",
     "Q charge/mA_h": "Q charge/mA.h",
     "Capacity/mA_h": "Capacity/mA.h",
-    "Efficiency/%": "Efficiency/%",
-    "control/V": "control/V",
-    "control/mA": "control/mA",
-    "cycle number": "cycle number",
-    "P/W": "P/W",
-    "R/Ohm": "R/Ohm",
-    "Rec#": "Rec#",
     "RecNo": "Rec#",
 }
+
+BIOLOGIC_COLUMN_MAPPING = {
+    **GENERIC_COLUMN_MAPPING,
+    **{"Ewe/V": "Ecell/V"},
+}
+
+MACCOR_COLUMN_MAPPING = {
```

Comment:
Can these column mappings be added by a user in the backend when they create a new parser?

Reply:
No they can't at the moment, which I agree is a shame. It would be nice for more flexibility and visibility. If I was going to do it I would do it by: 
- Creating a new `mapping` field on Parser (probably json or FK to totally separate model)
- Refactoring the existing code to apply the mappings, because the code in `parsing_engines/` doesn't have any knowledge of which `Parser` is being used. We'd probably have to pass it through from `parse_data_file` when it's called in the models.

I'd do it if we had more time. I'll open a separate issue for the above though so it could be tackled independently.